### PR TITLE
Document Flow Classifier concept

### PR DESCRIPTION
**Pull request by krdln**
Resolves: https://github.com/fluxninja/cloud/issues/5898

This change is [Reviewable](https://reviewable.io/reviews/fluxninja/aperture/338)

### CODE REVIEW

File: docs/content/concepts/flow-control/label/classifier.md [hunk 1]
```diff
@@ -3,5 +3,108 @@ title: Flow Classifier
...
+
+A classifier is defined as a resource in a [policy][policies]
+and describes a set of _rules_ on how to create new flow labels based
+on request metadata. Aperture uses Envoy's [External Authorization][ext-authz]
+definition to describe the request metadata (more specifically, the
+[AttributeContext][attr-context]). The [INPUT section at this Rego
+playground][rego-playground] is an example how the request attributes may look like.
+
+:::note
+Classifiers are available only on _traffic_ control points. That's because they
```
Comment (harjotgill) [inline docs/content/concepts/flow-control/label/classifier.md:21]: Labels can be provided directly by developers for feature control points.

Comment (krdln) [reply docs/content/concepts/flow-control/label/classifier.md:21]: Yeah, that's what I've written in next sentence. Do you think it needs some rewording?

Comment (harjotgill) [reply docs/content/concepts/flow-control/label/classifier.md:21]: It sounded like a limitation of feature control points — we can reword that features already have labels provided directly by developers for each request

Comment (krdln) [reply docs/content/concepts/flow-control/label/classifier.md:21]: Agreed. Will do

**Review by jaidesai-fn — APPROVED** [review]
> Reviewed 10 of 10 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/fluxninja/aperture/338)* status: all files reviewed, 1 unresolved discussion (waiting on @harjotgill)

**Review by hdkshingala — APPROVED** [review]

**Pull request merged**